lib/Cake/Model/Behavior/TranslateBehavior.php

```diff
@@ -74,9 +93,16 @@ public function setup(Model $Model, $config = array()) {
 			return false;
 		}
 
-		$this->settings[$Model->alias] = array();
+		$this->settings[$Model->alias] = array(
+			'fields' => array(),
+			'join_type' => 'INNER',
```

Comment:
Wouldn't this be 'joinType' to follow the naming conventions of other array keys?